[2023-03-08T06:18:25.244+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_postgrest_v3.insert_table manual__2023-03-08T06:17:16.528070+00:00 [queued]>
[2023-03-08T06:18:25.264+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: dag_postgrest_v3.insert_table manual__2023-03-08T06:17:16.528070+00:00 [queued]>
[2023-03-08T06:18:25.265+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-03-08T06:18:25.265+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 6
[2023-03-08T06:18:25.265+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-03-08T06:18:25.292+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): insert_table> on 2023-03-08 06:17:16.528070+00:00
[2023-03-08T06:18:25.300+0000] {standard_task_runner.py:55} INFO - Started process 11558 to run task
[2023-03-08T06:18:25.303+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_postgrest_v3', 'insert_table', 'manual__2023-03-08T06:17:16.528070+00:00', '--job-id', '102', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_postgres.py', '--cfg-path', '/tmp/tmp17ed7ax8']
[2023-03-08T06:18:25.306+0000] {standard_task_runner.py:83} INFO - Job 102: Subtask insert_table
[2023-03-08T06:18:25.393+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_postgrest_v3.insert_table manual__2023-03-08T06:17:16.528070+00:00 [running]> on host df4d96e1a045
[2023-03-08T06:18:25.519+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Aditya
AIRFLOW_CTX_DAG_ID=dag_postgrest_v3
AIRFLOW_CTX_TASK_ID=insert_table
AIRFLOW_CTX_EXECUTION_DATE=2023-03-08T06:17:16.528070+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-08T06:17:16.528070+00:00
[2023-03-08T06:18:25.520+0000] {sql.py:254} INFO - Executing: 
        insert into dag_run (dt,dag_id) values('2023-03-08','dag_postgrest_v3')
        
[2023-03-08T06:18:25.545+0000] {base.py:73} INFO - Using connection ID 'postgrest_localhost' for task execution.
[2023-03-08T06:18:25.959+0000] {base.py:73} INFO - Using connection ID 'postgrest_localhost' for task execution.
[2023-03-08T06:18:25.965+0000] {sql.py:375} INFO - Running statement: 
        insert into dag_run (dt,dag_id) values('2023-03-08','dag_postgrest_v3')
        , parameters: None
[2023-03-08T06:18:25.966+0000] {sql.py:384} INFO - Rows affected: 1
[2023-03-08T06:18:25.980+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=dag_postgrest_v3, task_id=insert_table, execution_date=20230308T061716, start_date=20230308T061825, end_date=20230308T061825
[2023-03-08T06:18:26.038+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-03-08T06:18:26.071+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
